tests/features/contact_form/submit_contact_form.feature

```diff
@@ -28,7 +28,9 @@ Feature: Submit the contact form
     # The honeypot field that needs to be empty on submission.
     Then the following fields should be present "user_homepage"
     And I should see the text "Submissions of this form are processed by a contractor of the European Commission."
+    And I am on the homepage
 
+    Then I click "Contact Joinup Support" in the "Footer" region
```

Comment:
We need more test coverage **but check if is not yet there**:

* On homepage with `?destination=...` already in URL
* An arbitrary page with `?destination=...` already in URL
* Homepage w/o `?destination=...`
* Arbitrary page w/o `?destination=...`